src/components/SearchForm/SearchForm.js

```diff
@@ -1,13 +1,34 @@
-import './SearchForm.css';
-
-function SearchForm(props){
-    return (
-        <form className='search-form' action='#' name='searchForm'>
-        <input type='text' name="keywords" placeholder='Enter topic' className='search-form__input'/>
-        <button type='submit' className='search-form__submit-button'>Search</button>
-
-        </form>
-    )
+import './SearchForm.css'
+import { useState } from 'react'
 
+function SearchForm({ onSearch }) {
+  const [keyword, setKeyword] = useState('')
+  const handleChange = (evt) => {
+    setKeyword(evt.target.value)
+  }
+  const handleSubmit = (evt) => {
+    evt.preventDefault()
+    onSearch(keyword)
+  }
+  return (
+    <form
+      className="search-form"
+      action="#"
+      name="searchForm"
+      onSubmit={handleSubmit}
+    >
+      <input
+        type="text"
+        onChange={handleChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components